testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js

```diff
@@ -9,15 +9,26 @@ QUnit.testStart(function() {
         height: 440px;\
     }\
 </style>\
-<div style="padding: 0px 40px; margin: 0px 50px">\
+<div id="testWrapper">\
     <div id="testContainer"></div>\
 </div>\
 <div id="root">\
     <div id="container" class="dx-datagrid dx-widget"></div>\
 </div>\
-<div id="itemsContainer"><div style="width:125px; display: inline-block;" ></div><div style="width:125px; display: inline-block;" ></div></div>';
+<div id="itemsContainer"><div></div><div></div></div>';
```

Comment:
Let's make formatting more obvious and add classes to inner divs:

```suggestion
<div id="itemsContainer">
  <div class="itemsContainer__child"></div>
  <div class="itemsContainer__child"></div>
</div>
`;
```